src/main/index.js

```diff
@@ -0,0 +1,11 @@
+import { NumberOfProductsAndCategories } from '../numberOfProductsAndCategories'
+import { products } from '../helpers/products'
+import styles from './styles.css'
+import { ProductInformation } from '../productInformation'
+
+export const Main = () => (
+  <main className={styles.main}>
+    <NumberOfProductsAndCategories name='Nikita' products={products} />
```

Comment:
it's something like `ProductsInfo` or `GeneralInfo` or just `Info`

Reply:
Thank you for name. I change it